### PR TITLE
meta field added to env model

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: models/models_methods.go [hunk 1]
```diff
@@ -13,19 +13,22 @@ import (
 const (
 	// OptionsKey ...
 	OptionsKey = "opts"
+	// MetaKey ...
+	MetaKey = "meta"
```
Comment (viktorbenei) [inline models/models_methods.go]: remove

Comment (godrei) [inline models/models_methods.go]: remove this, if this is not used

File: models/models.go [hunk 2]
```diff
@@ -14,6 +14,8 @@ type EnvironmentItemOptionsModel struct {
 	IsRequired        *bool    `json:"is_required,omitempty" yaml:"is_required,omitempty"`
 	IsDontChangeValue *bool    `json:"is_dont_change_value,omitempty" yaml:"is_dont_change_value,omitempty"`
 	IsTemplate        *bool    `json:"is_template,omitempty" yaml:"is_template,omitempty"`
+	//
+	Meta *map[string]interface{} `json:"meta,omitempty" yaml:"meta,omitempty"`
```
Comment (viktorbenei) [inline models/models.go]: pointer (`*`) not required

File: models/models_methods.go [hunk 3]
```diff
@@ -270,6 +282,9 @@ func (env *EnvironmentItemModel) FillMissingDefaults() error {
 	if options.SkipIfEmpty == nil {
 		options.SkipIfEmpty = pointers.NewBoolPtr(DefaultSkipIfEmpty)
 	}
+	if options.Meta == nil {
+		options.Meta = pointers.NewMapStringInterfacePtr(map[string]interface{}{})
```
Comment (viktorbenei) [inline models/models_methods.go]: not required, simply `map[string]interface{}{}` to init

File: models/models_methods_test.go [hunk 4]
```diff
@@ -117,6 +117,24 @@ func TestParseFromInterfaceMap(t *testing.T) {
...
+
+	t.Log("parse meta field")
+	{
+		serializedObj := `key: "value"`
+		var obj interface{}
+		require.NoError(t, yaml.Unmarshal([]byte(serializedObj), &obj))
+
+		model := map[string]interface{}{}
+		model["meta"] = obj
+		require.NoError(t, envOptions.ParseFromInterfaceMap(model))
```
Comment (viktorbenei) [inline models/models_methods_test.go]: check **value** too, not just Error

**Pull request merged**